src/cozmopy/cozmopy.cpp

```diff
@@ -1,51 +1,129 @@
+        .def("createState", &Cozmo::createState, py::arg("x"), py::arg("y"), py::arg("th"))
+        .def("createInterpolatedTraj", &Cozmo::createInterpolatedTraj, py::arg("waypoints"))
+        .def("executeTrajectory", [](
+            libcozmo::Cozmo& cozmo,
+            const std::chrono::duration<double, std::milli> milliseconds,
+            aikido::trajectory::TrajectoryPtr traj) {
+            return cozmo.executeTrajectory(
+                std::chrono::duration_cast<std::chrono::milliseconds>(milliseconds), traj);
+        })
         .def("setState", [](
             libcozmo::Cozmo& cozmo,
-            const double& x, 
-            const double& y, 
-            const std::vector<double>& quat)        
-        {
+            const double& x,
+            const double& y,
+            const std::vector<double>& quat) {
             Eigen::Quaterniond q(quat[0], quat[1], quat[2], quat[3]);
             cozmo.setState(x, y, q);
         });
-        
+
         py::class_<libcozmo::Waypoint>(m, "Waypoint")
-		.def(py::init([](const double x, const double y, const double th, const double t) {
-			libcozmo::Waypoint w = {.x = x, .y = y, .th = th, .t = t};
-			return w;
-		}));
+        .def(py::init([](const double x, const double y, const double th, const double t) {
+            libcozmo::Waypoint w = {.x = x, .y = y, .th = th, .t = t};
+            return w;
+        }));
+
+    py::class_<actionspace::ObjectOrientedActionSpace>(m, "ObjectOrientedActionSpace")
+        .def(py::init<const std::vector<double>&,
+                      const std::vector<double>&,
+                      const Eigen::Vector2d&,
+                      const Eigen::Vector2d&,
+                      const int&>())
+        .def("action_similarity", [](
+            const actionspace::ObjectOrientedActionSpace& actionspace,
+            const int& action_id1,
+            const int& action_id2) {
+            double similarity = -1;
+            bool successful =
+                actionspace.action_similarity(action_id1, action_id2, &similarity);
+            return similarity, successful;
+        })
+        .def("get_action",
+            &actionspace::ObjectOrientedActionSpace::get_action,
+            py::arg("action_id"))
+        .def("is_valid_action_id",
+            &actionspace::ObjectOrientedActionSpace::is_valid_action_id,
+            py::arg("action_id"))
+        .def("get_generic_to_object_oriented_action", [](
+            const actionspace::ObjectOrientedActionSpace& actionspace,
+            const int& action_id,
+            const aikido::statespace::StateSpace::State& _state){
+            actionspace::ObjectOrientedActionSpace::CozmoAction action(
+                0.0, Eigen::Vector3d(0, 0, 0));
+            bool successful = actionspace.get_generic_to_object_oriented_action(
+                action_id, _state, &action);
+            return action;
+        })
+        .def("publish_action",
```

Comment:
We temporarily removed this from the master branch because the build was failing so you should remove this for now